docs/outputs.rst

```diff
@@ -140,13 +140,20 @@ mid-thickness surface mesh::
 
 Surface output spaces include ``fsnative`` (full density subject-specific mesh),
 ``fsaverage`` and the down-sampled meshes ``fsaverage6`` (41k vertices) and
-``fsaverage5`` (10k vertices, default).
+``fsaverage5`` (10k vertices, default). Additionally, the BOLD can be resampled to
+``fsLR`` surfaces, a hybrid version of the ``fsaverage`` atlas with both hemispheres
+in precise geographic correspondence, with varying mesh densities: 32k (default), 59k, and 164k.
 
 **Grayordinates files**.
 CIFTI is a container format that holds both volumetric (regularly sampled in a grid)
 and surface (sampled on a triangular mesh) samples.
-Sub-cortical time series are volumetric (supported spaces: ``MNI152NLin2009cAsym``), while cortical
-time series are sampled on the surface (supported spaces: ``fsaverage5``, ``fsaverage6``).
+Sub-cortical time series are volumetric (supported spaces: ``MNI152NLin6Asym``), while cortical
+time series are sampled to surface (supported spaces: ``fsLR``).
+By default, CIFTI outputs produced match up to `HCP Pipelines`_ low resolution CIFTI 
+output (91282 grayordinates).
+A higher resolution output can be created by increasing the surface density of ``fsLR`` to 59k
+adding the ``fsLR:den-59k`` to the list of spaces in the ``--output-spaces``
```

Comment:
It feels a bit weird that adding `fsLR:den-59k` will generate grayordinates of 168k (I believe). Although the surfaces will be den-59k. We might want to allow the density to be stated, such as `--cifti-output 91k` and then have the `--output-spaces` work independently. Mapping:

`--cifti-output` -> generates 91k
`--cifti-output 91k` -> generates 91k
`--cifti-output 168k` -> generates 168k
and so on so forth.

WDYT?

Reply:
Yeah, it might be best to tie the CIFTI resolution to its own flag 😄

I like the idea, will patchwork that in and then clean it up in #1892 (which I hope to address right after this is merged)